packages/replay/src/types.ts

```diff
@@ -113,10 +113,70 @@ export interface ReplayPluginOptions extends SessionOptions {
   }>;
 }
 
+export interface ReplayIntegrationPrivacyOptions {
+  /**
+   * Mask text content for elements that match the CSS selectors in the list.
+   */
+  mask?: string[];
+
+  /**
+   * Unmask text content for elements that match the CSS selectors in the list.
+   */
+  unmask?: string[];
+
+  /**
+   * Block elements that match the CSS selectors in the list. Blocking replaces
+   * the element with an empty placeholder with the same dimensions.
+   */
+  block?: string[];
+
+  /**
+   * Unblock elements that match the CSS selectors in the list. This is useful when using `blockAllMedia`.
+   */
+  unblock?: string[];
+
+  /**
+   * Ignore input events for elements that match the CSS selectors in the list.
+   */
+  ignore?: string[];
```

Comment:
should we go with `ignoreEvents` here?

Reply:
@mydea hmm my worry is that people will confuse it for DOM events e.g. clicks?